runtime/runtime_controller.cc

```diff
@@ -272,7 +272,7 @@ bool RuntimeController::DispatchPointerDataPacket(
     const PointerDataPacket& packet) {
   if (auto* platform_configuration = GetPlatformConfigurationIfAvailable()) {
     TRACE_EVENT0("flutter", "RuntimeController::DispatchPointerDataPacket");
-    platform_configuration->get_window(0)->DispatchPointerDataPacket(packet);
```

Comment:
On the framework side, how does it know which window to associate the pointer data with?  Or does it just not do that, and pointer data is just in global screen coordinates?

Reply:
We'll be adding a `viewId` member to `PointerData` that will be set by the embedder. However, that requires changing the embedder API which we'll do only after we've reviewed the multi-view engine spec.